### DISCUSSION TITLE
Lesson 2.6 Not in Slides

### DISCUSSION THREAD

**Discussion by prsmendonca** [TEALSK12/introduction-to-computer-science#339] [General]
The slides in https://github.com/TEALSK12/introduction-to-computer-science/tree/master/slidedecks skip from 2.5 to 3.1. I later found out that Lesson 2.6 is just the Pong project, and now I see that lesson_26.md is in the repo, but before that I made a big mess in my sequence of classes by skipping from 2.5 to 3.1. I won't make this mistake again - i.e., I'll no longer follow the slides - but would it be a good idea to have a Lesson 2.6 slide (an perhaps a Lab 2.6 in the Unit 2 Word and PDF directories), if only too keep things more uniform?

**Comment by DBashirTEALS** [comment]
Hello!  Thanks for bringing this to our attention.  I'll reach out to the curriculum team to investigate this further.

Dr. Bashir

**Comment by jdonwells** [comment]
I ended up creating my own set of tasks to substitute for the do now tasks during the 2.6 Pong project. I took a game with similarities to the Pong game and worked through it in 15 minute bites each day. The game is Breakout. Every day I give the students a starter project. I then start there and finish one specific thing. For example, day one was moving a paddle back and forth. It seemed helpful.
